content/docs/usage.md

```diff
@@ -99,9 +114,9 @@ cat results.txt >> report.md
 
 🖼️ **Images** Display images using the markdown or HTML. Note that if an image
 is an output of your ML workflow (i.e., it is produced by your workflow), you
-will need to use the `cml-publish` function to include it a CML report. For
+will need to use the `cml publish` command to include it a CML report. For
 example, if `graph.png` is output by `python train.py`, run:
 
 ```bash
-cml-publish graph.png --md >> report.md
+cml publish graph.png --md >> report.md
```

Comment:
`report.md` vs `myreport.md` 🙃

Reply:
I also prefer ```report.md``` @casperdcl

Reply:
For me is a matter of consisntency, not preference, but I wholeheartedly agree with @DavidGOrtega. No matter which name we settle with, it should be used everywhere.

In my opinion, `report.md` looks cleaner. Is the `my` prefix intended to denote a customizable value versus some magic keyword for shell–blind users? 🤔

Reply:
made consistent now. The "my" is indeed to signal customisation. In the full yaml workflows, it is not necessary since it is obvious from the `echo "# Title" >> (my)report.md` that it's user-generated. However in `/doc/ref/*` we don't (yet) have full workflows so it's unclear where `report.md` comes from; hence the `my` prefix.

Reply:
Sounds fair. It looks like this could be a deeper problem with CML “wet” (i.e. non–DRY) approach to report building. 🤔  Our recommended approach is definitely easy to understand, but we might want to use a more template–ish approach in the future. 🤷🏼 

#### Original
```bash
cml image --md confusion_matrix.png >> [my]report.md
cml pr --md dvc.lock >> [my]report.md
cml comment [my]report.md
```

#### Indented `heredoc`
```bash
cml comment - <<-END
        # Report
        $(cml image --md confusion_matrix.png)
        $(cml pr --md dvc.lock)
END
```

#### Pipes and in–place append (suboptimal)
```bash
echo "# Report" | cml comment --append -
cml image --md confusion_matrix.png | cml comment --append -
cml pr --md dvc.lock | cml comment --append -
```

Reply:
Anyhow, if `[my]report.md` is going to be a placeholder, wouldn't it be better to use a true BNF–ish placeholder like `<file>` and leave appending approaches to full examples or an exercise for the end user? 🤔

Reply:
scrubbed ~my~